packages/host/app/services/matrix-service.ts

```diff
@@ -71,6 +73,7 @@ export type OperatorModeContext = {
   submode: Submode;
   openCardIds: string[];
 };
+
```

Comment:
Do you know if you added this manually or it was an autoformat thing? I wonder if we need to tweak some lint/formatter settings, I can’t imagine not wanting a space here 🤔

Reply:
I must have added it manually. Tried to save the file without the space, it saves fine.